sim/src/simulator.rs

```diff
@@ -28,12 +29,14 @@ pub struct Simulator {
 
 impl Simulator {
     pub fn new(seed: u64) -> Self {
-        Simulator {
+        let mut sim = Simulator {
             elements: HashMap::new(),
             graph: Graph::new(),
             node_index_to_node: HashMap::new(),
             seed,
-        }
+        };
+        sim.add_node(SINK_NAME, 1, 0, 0, None);
```

Comment:
Why is this needed? I though you could just set the generation rate of ratings and details to 0?

Reply:
Generation rate is how many RPCs are created out of thin air, not how many are passed on.  The egress rate is how many are passed on, but I don't think it's a good idea to set it to 0, because we're modeling a response being sent back as well.  So an RPC can optionally go back up the tree.

Reply:
Based on my comments in the other thread I would say both values should be set to 0 in ratings and details and this extra sink should be removed. We do not have a model of request and response yet. That we will implement soon. Let's assume this just models the request dag that ends there.

Reply:
Okay, I changed it back.